ehrapy/api/preprocessing/_data_imputation.py

```diff
@@ -5,11 +5,12 @@
 from anndata import AnnData
 from rich import print
 from rich.progress import Progress, SpinnerColumn
-from sklearn.ensemble import ExtraTreesRegressor, RandomForestClassifier
 from sklearn.experimental import enable_iterative_imputer  # noqa: F401
-from sklearn.impute import IterativeImputer, KNNImputer, SimpleImputer
```

Comment:
are those two removed?

Reply:
Locally imported after the Intel sklearn patch.